src/components/Login.tsx

```diff
@@ -18,10 +21,10 @@ const Login: React.FC = () => {
   const signUpLink = isSignUp ? "Sign in now" : "Sign up now";
   console.log("page = ", signBtn);
   function handleLoginPage() {
-    setIsSignUp(prevVal => !prevVal);
+    setIsSignUp((prevVal) => !prevVal);
```

Comment:
this can be written as setIsSignUp(!prevVal)

Reply:
old logic is better